hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeCreateRequest.java

```diff
@@ -164,7 +164,7 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
           // Replay implies the response has already been returned to
           // the client. So take no further action and return a dummy
           // OMClientResponse.
-          LOG.debug("Replayed Transaction {} ignored. Request: {}",
+          LOG.info("Replayed Transaction {} ignored. Request: {}",
```

Comment:
Keeping this log message at info level will flood the log file. It prints the Request body also.